tensorflow/python/framework/tensor_util_test.py

```diff
@@ -662,6 +663,43 @@ def testShapeEquals(self):
     self.assertFalse(tensor_util.ShapeEquals(t, [1, 4]))
     self.assertFalse(tensor_util.ShapeEquals(t, [4]))
 
+  def testSeries(self):
+    df = pd.Series([10, 20, 30])
```

Comment:
I would suggest testing this with a dummy class that implements `__array__` rather than actually using pandas.